deploy/chart/ingressmonitorcontroller/README.md

```diff
@@ -0,0 +1,94 @@
+- [StatusCake](https://www.statuscake.com) ([Additional Config](https://github.com/stakater/IngressMonitorController/blob/master/docs/statuscake-configuration.md))
+
+## Usage
+
+The following quickstart let's you set up Ingress Monitor Controller to register uptime monitors for ingresses/routes in all namespaces:
+
+1. Update the `values.yaml` and set the following properties
+
+| Key           | Description                                                               | Example                            | Default Value                      |
+|---------------|---------------------------------------------------------------------------|------------------------------------|------------------------------------|
+| useFullName          | Option to use full name containing release name for resources                                                      | `false`                        | `false`                        |
+| matchLabels          | Match labels for deployment selector                                                      | `{}`                        | `{}`
+| deployment.labels          | Labels for deployment                                                      | `{}`                        | `{}`
+| deployment.annotations           | Annotations for deployment                                                      | `{}`                        | `{}`
+| config.labels          | Labels for configmap                                                      | `{}`                        | `{}`
+| config.annotations          | Annotations for configmap                                                      | `fabric8.io/target-platform: kubernetes`                        | `fabric8.io/target-platform: kubernetes`
+| rbac.create          | Option to create rbac resources                                                      | `true`                        | `true`
+| rbac.labels          | Labels for rbac                                                      | `{}`                        | `{}`
+| serviceAccount.create          | Option to create service account                                                      | `true`                        | `true`
+| serviceAccount.labels          | Labels for serviceAccount                                                      | `{}`                        | `{}`
+| serviceAccount.name          | Name of serviceAccount                                                      | `ingressmonitorcontroller`                        | `ingressmonitorcontroller`
+| image.name          | Image of ingressMonitorController                                                      | `stakater/ingressmonitorcontroller`                        | `stakater/ingressmonitorcontroller`
+| image.tag          | Version of ingressMonitorController Image                                                      | `1.0.47`                        | `1.0.47`
```

Comment:
Please sync the default value with the values.yaml file.
`2.0.2` in values.yaml file now